lang/src/accounts/unchecked_account.rs

```diff
@@ -12,18 +12,18 @@ use std::ops::Deref;
 /// Explicit wrapper for AccountInfo types to emphasize
 /// that no checks are performed
 #[derive(Debug, Clone)]
-pub struct UncheckedAccount<'info>(AccountInfo<'info>);
+pub struct UncheckedAccount<'info>(&'info AccountInfo<'info>);
```

Comment:
What do you think about changing all owned `AccountInfo`'s in the other account types to references as well? We also create them by cloning internally.